<a href="https://yeamanlab.weebly.com"><img src="https://img.shields.io/badge/yeaman-lab-blue"></a>
[![](https://img.shields.io/badge/CoAdapTree-UBC-green.svg?style=popout&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAXCAYAAAAP6L+eAAAGPklEQVR42oVVa0xb5xl2GtIWyKLcWNJAFgLkYjAJN2McTGzwBWMcwDYmELCxjS8Y29jG2CGYgB1ICAFMoLZbaGi4tKVlbdN0yySyTkzVqqptpE77MVXrqm5Sf6Stum7LUjIwfnbOYbGy9fZJn877fDrv8z3v5byH9n3L5/M9Ru5Hz25cG7EHu81hwowj8fh5+6CjVvTxkF3XsjQ3l/jQ7zsJQaNtIvdDfHM+lLG8cD2VtEe6zHOBdjVkR7ZXk1gtLnnLLOFAK+Lesym1SeTZDypVKpWbcefOFhK//YtZ/a35ib/pC2kHFSlJEp2oBNLczD+9feO6bahD/1tB2h7wc47PLw13JgbPmqp/+dyIfHl5OS5GCGCTzVbxBGnfvbuU+NK4byHs63C89fIz0lB/x2eT/S7MX/a+qmQd+qgoaQfCPhemfA7Iio5BUpj7Gx2f/d6QpQkvXR34ZDkU2vpflbzYDdcHXcxRtzH8tM/51evhi/Bo5O+JGRmLAw4TFif80IvZoO9OWi3eRlsRZ6VFuFn0dS49HSpuPqwSzu1vpUCj4T05PeLjTF10f2hTlsOlrf/HZH/HvzlH9kPKzoO0gL4iL2FG1GJulJd5CGSQ++O3oTTrMLS8vJUxhwqvhC/9udeqnhrzGL1kTim1JiU/09Ik//v5dn1EVHRslbF/H5qlZdFG8clIfnoKspP34sDWHdG9NFpUzmHCUHcKEjaRhpwjqM6nw35GCq+1JdJySvjJ1ICnjVLb2yqXhfyOj5Ri3ppMwEV4oDOSsisxSqoK++wI+azrpM1Iy8CeXT+FmH4AV7vMmBnxwtumBouRtZ6Z8BiaRex7MwFfP4A4Gj7+1RN97drpZ/rd6LU0/YtoQlz12tDXWkeFWysuhd+mgqy0kMLpKSlgpiShrpQFS6MMamkZznCZaBEVRoIXXKsvjvYg1N3morkNCl2PVYd6qfCBXS1HIYOOeNrjOCM6gYL0VOhrJWiuEVCkGfuSsT9pN2FvofA+YnPpB6E4wVhXCVnwG+twxanpnpz0JdC6nLq09pbGtdOVQrRrG9areCz02dSkI7VNtRWYGfIQOa2gcNrO3ajiFkHAzqMwUaC10uPpqOMXvt9axffEumFpaS7xxvzUXUODDM8OdEVaifAGnHpcsGtIR0qdjM/BOcNp8kmd8fMzcdmpQ+BcK9z601FJUQ7Kcw9/rRIw/znh63wDwOMkd/zrLz7/l/H+bggLsiMefRNxmoi2+irI+cXYEb8DOxO3xyJIT06hnsU52QRpA6wqZbQk+yhOsY6tXnFqMRvwfQBgM6U6OHzxg/GBHuhqJat85nHwWXnIz8zYIHoqGU51DTq1MvwkPgFxRAS5aakoYaQjaw8NPyParzz/CExEHbxG1VcTHltKLB0vhIY//PXCc5gd9a8G/W6Y6iQY7zGjumyjE8pZxzFy1oBhIvQGKZ+aUwo+G4GzmtXZKx1E2rS3LHJBuEejkDwcDxTxWHf75PQFN9qbFCt9NgM8RhXamhQgphlExfkU+eHkA2iWSwjMovCB+Cehkp78JnDOiMlBzzvvL9/aS3IZjcYtMcU3F6b3zQ/3RvosWuyM27LOyqJTzjlHj6KMzcTWhF0QFeXBUi8hohFAITwRZdIzotycozDW8BDqs+L6Vb8yRkiO3eVpa8HTVoEv1G37YuK8C0Nu81qvuRHlnLyojFeA7NSNwiUQBTUoxPDbVetk59Rw2WsqSckdl15+by7QjVenAq2PTkraH98cGLs9bsBsV/V9XY0IUg4rcqnT/GDQY4ZbI8egU422hnJkH0olCrc5WlvOQX1l8Te6ah7Om1RjAZ+p4I2ZUcvt114o/h/F938/9+bnvwtjtpONUUfj140VG1+ZgM26z6BnPpCVnYh6WxuiNeVcanYohJxov0ON4AU7nr3cg59fC/JJpm/9lvDpje2fvXut55VhzQKLRts27m1vbpCUfmkhcqcSbHxdVSUsWM9UrpvqK1cqeRzUivh/9Ts1706N9PzhtZngSZJncVG5mfZjazZ4adewXWUYczTe1FfzoweT9kSU/CK4tAqYlBUQF+aOfofbpkcBcdNi7Cby1v9/wdlYnV2Wz0ivFZbwLUrhO72tdZ9fdrWcjJF9z/oPOHr50Rw5rjUAAAAASUVORK5CYII=)](http://coadaptree.forestry.ubc.ca)
<a href="https://twitter.com/iowensis"><img src="https://img.shields.io/twitter/follow/iowensis?color=green&logoColor=blue&style=social"></a>

# Brandon Lind
Overview of work done during time with the Yeaman Lab. More detailed descriptions accompany the linked repositories' READMEs.

-----
### Contact Info
personal email: lind.brandon.m@gmail.com

academic email: brandon.lind@ubc.ca

github: https://www.github.com/brandonlind

research page: https://brandonlind.github.io

-----
### Usage and license information

If you use or are inspired by code in this repository please cite the following work or contact me about how to cite. 

-----
### Repositories completed or in progress (publications coming soon)
1. [CoAdapTree CMH test](https://github.com/brandonlind/cmh_test) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083798.svg)](https://doi.org/10.5281/zenodo.5083798)
    - This code takes output from the VarScan pipeline and calculates Cochran-Mantel-Haenszel chi-squared tests on stratified contingency tables of read counts standardized by population ploidy.
```
Lind B (2021) GitHub.com/brandonlind/cmh_test: preprint release (Version 1.0.0). Zenodo. http://doi.org/10.5281/zenodo.5083798
```

1. [Douglas-fir GEA populations (in progress)](https://github.com/brandonlind/doug_fir_mapping_pops_analyses) 
    - This repository contains the processessing scripts needed to analyze range-wide populations of Douglas-fir.

1. [Jack pine GEA populations (in progress)](https://github.com/brandonlind/jack_pine_mapping_pops_analyses)
    - This repository contains the processessing scripts needed to analyze range-wide populations of jack pine.

1. [CoAdapTree Douglas-fir / lodgepole pine SNP array (in progress)](https://github.com/brandonlind/coadaptree_snparray)
    - This repository covers the code used to develop a 50k SNP array split evenly between Douglas-fir and lodgepole pine.

-----
### Publications and associated repositories
1. (accepted) BM Lind*, M Lu*, D Vidakovic, P Singh, T Booker, S Yeaman, S Aitken. Haploid, diploid, and pooled exome capture recapitulate features of biology and paralogy in two non-model tree species. Molecular Ecology Resources. Molecular Ecology Resources - [preprint available at bioRxiv](https://doi.org/10.1101/2020.10.07.329961)
    - <b> Abstract </b>
Despite their suitability for studying evolution, many conifer species have large and repetitive giga-genomes (16-31Gbp) that create hurdles to producing high coverage SNP datasets that captures diversity from across the entirety of the genome. Due in part to multiple ancient whole genome duplication events, gene family expansion and subsequent evolution within Pinaceae, false diversity from the misalignment of paralog copies create further challenges in accurately and reproducibly inferring evolutionary history from sequence data. Here, we leverage the cost-saving benefits of pool-seq and exome-capture to discover SNPs in two conifer species, Douglas-fir (Pseudotsuga menziesii var. menziesii (Mirb.) Franco, Pinaceae) and jack pine (Pinus banksiana Lamb., Pinaceae). We show, using minimal baseline filtering, that allele frequencies estimated from pooled individuals show a strong positive correlation with those estimated by sequencing the same population as individuals (r > 0.948), on par with such comparisons made in model organisms. Further, we highlight the use of haploid megagametophyte tissue in identifying sites that are likely due to misaligned paralogs. Together with additional minor filtering, we show that it is possible to remove many of the loci with large frequency estimate discrepancies between individual and pooled sequencing approaches, improving the correlation further (r > 0.973). Our work addresses bioinformatic challenges in non-model organisms with large and complex genomes, highlights the use of megagametophyte tissue for the identification of paralog sites when sequencing large numbers of populations, and suggests the combination of pool-seq and exome capture to be robust for further evolutionary hypothesis testing in these systems.
    - [VarScan autonomous SNP calling pipeline](https://github.com/brandonlind/varscan_pipeline) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083302.svg)](https://doi.org/10.5281/zenodo.5083302)
    - [Code for manuscript](https://github.com/brandonlind/testdata_validation) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083292.svg)](https://doi.org/10.5281/zenodo.5083292)
    - [Data for manuscript](https://doi.org/10.5061/dryad.k0p2ngf7w)
[![DOI](https://zenodo.org/badge/DOI/10.5061/dryad.k0p2ngf7w.svg)](https://doi.org/10.5061/dryad.k0p2ngf7w)

