<a href="https://yeamanlab.weebly.com"><img src="https://img.shields.io/badge/yeaman-lab-blue"></a>
[![](https://img.shields.io/badge/CoAdapTree-UBC-green.svg?style=popout&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAXCAYAAAAP6L+eAAAGPklEQVR42oVVa0xb5xl2GtIWyKLcWNJAFgLkYjAJN2McTGzwBWMcwDYmELCxjS8Y29jG2CGYgB1ICAFMoLZbaGi4tKVlbdN0yySyTkzVqqptpE77MVXrqm5Sf6Stum7LUjIwfnbOYbGy9fZJn877fDrv8z3v5byH9n3L5/M9Ru5Hz25cG7EHu81hwowj8fh5+6CjVvTxkF3XsjQ3l/jQ7zsJQaNtIvdDfHM+lLG8cD2VtEe6zHOBdjVkR7ZXk1gtLnnLLOFAK+Lesym1SeTZDypVKpWbcefOFhK//YtZ/a35ib/pC2kHFSlJEp2oBNLczD+9feO6bahD/1tB2h7wc47PLw13JgbPmqp/+dyIfHl5OS5GCGCTzVbxBGnfvbuU+NK4byHs63C89fIz0lB/x2eT/S7MX/a+qmQd+qgoaQfCPhemfA7Iio5BUpj7Gx2f/d6QpQkvXR34ZDkU2vpflbzYDdcHXcxRtzH8tM/51evhi/Bo5O+JGRmLAw4TFif80IvZoO9OWi3eRlsRZ6VFuFn0dS49HSpuPqwSzu1vpUCj4T05PeLjTF10f2hTlsOlrf/HZH/HvzlH9kPKzoO0gL4iL2FG1GJulJd5CGSQ++O3oTTrMLS8vJUxhwqvhC/9udeqnhrzGL1kTim1JiU/09Ik//v5dn1EVHRslbF/H5qlZdFG8clIfnoKspP34sDWHdG9NFpUzmHCUHcKEjaRhpwjqM6nw35GCq+1JdJySvjJ1ICnjVLb2yqXhfyOj5Ri3ppMwEV4oDOSsisxSqoK++wI+azrpM1Iy8CeXT+FmH4AV7vMmBnxwtumBouRtZ6Z8BiaRex7MwFfP4A4Gj7+1RN97drpZ/rd6LU0/YtoQlz12tDXWkeFWysuhd+mgqy0kMLpKSlgpiShrpQFS6MMamkZznCZaBEVRoIXXKsvjvYg1N3morkNCl2PVYd6qfCBXS1HIYOOeNrjOCM6gYL0VOhrJWiuEVCkGfuSsT9pN2FvofA+YnPpB6E4wVhXCVnwG+twxanpnpz0JdC6nLq09pbGtdOVQrRrG9areCz02dSkI7VNtRWYGfIQOa2gcNrO3ajiFkHAzqMwUaC10uPpqOMXvt9axffEumFpaS7xxvzUXUODDM8OdEVaifAGnHpcsGtIR0qdjM/BOcNp8kmd8fMzcdmpQ+BcK9z601FJUQ7Kcw9/rRIw/znh63wDwOMkd/zrLz7/l/H+bggLsiMefRNxmoi2+irI+cXYEb8DOxO3xyJIT06hnsU52QRpA6wqZbQk+yhOsY6tXnFqMRvwfQBgM6U6OHzxg/GBHuhqJat85nHwWXnIz8zYIHoqGU51DTq1MvwkPgFxRAS5aakoYaQjaw8NPyParzz/CExEHbxG1VcTHltKLB0vhIY//PXCc5gd9a8G/W6Y6iQY7zGjumyjE8pZxzFy1oBhIvQGKZ+aUwo+G4GzmtXZKx1E2rS3LHJBuEejkDwcDxTxWHf75PQFN9qbFCt9NgM8RhXamhQgphlExfkU+eHkA2iWSwjMovCB+Cehkp78JnDOiMlBzzvvL9/aS3IZjcYtMcU3F6b3zQ/3RvosWuyM27LOyqJTzjlHj6KMzcTWhF0QFeXBUi8hohFAITwRZdIzotycozDW8BDqs+L6Vb8yRkiO3eVpa8HTVoEv1G37YuK8C0Nu81qvuRHlnLyojFeA7NSNwiUQBTUoxPDbVetk59Rw2WsqSckdl15+by7QjVenAq2PTkraH98cGLs9bsBsV/V9XY0IUg4rcqnT/GDQY4ZbI8egU422hnJkH0olCrc5WlvOQX1l8Te6ah7Om1RjAZ+p4I2ZUcvt114o/h/F938/9+bnvwtjtpONUUfj140VG1+ZgM26z6BnPpCVnYh6WxuiNeVcanYohJxov0ON4AU7nr3cg59fC/JJpm/9lvDpje2fvXut55VhzQKLRts27m1vbpCUfmkhcqcSbHxdVSUsWM9UrpvqK1cqeRzUivh/9Ts1706N9PzhtZngSZJncVG5mfZjazZ4adewXWUYczTe1FfzoweT9kSU/CK4tAqYlBUQF+aOfofbpkcBcdNi7Cby1v9/wdlYnV2Wz0ivFZbwLUrhO72tdZ9fdrWcjJF9z/oPOHr50Rw5rjUAAAAASUVORK5CYII=)](http://coadaptree.forestry.ubc.ca)
<a href="https://twitter.com/iowensis"><img src="https://img.shields.io/twitter/follow/iowensis?color=green&logoColor=blue&style=social"></a>

# Brandon Lind
Overview of work done during time with Sally Aitken and Sam on the CoAdapTree project. More detailed descriptions accompany the linked repositories' READMEs.

-----
### Contact Info

‚úâÔ∏è [lind.brandon.m@gmail.com](lind.brandon.m@gmail.com)\
üê¶ [@iowensis](https://www.twitter.com/iowensis)\
üéì [google scholar](https://scholar.google.com/citations?user=w0vu_LkAAAAJ&hl=en&oi=ao)\
üî¨ [research page](https://brandonlind.github.io)\
üîó [LinkedIn](https://www.linkedin.com/in/brandonlind-phd/)

-----
### Usage and license information

If you use or are inspired by code in this repository please cite the following work or contact me about how to cite. 

-----
### Repositories completed or in progress (publications coming soon)
1. [CoAdapTree pool-seq VarScan pipeline](https://github.com/coadaptree/varscan_pipeline) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083302.svg)](https://doi.org/10.5281/zenodo.5083302)
    - autonomous SNP calling pipeline for pool-seq data with VarScan
    ```
    Lind B (2021) GitHub.com/CoAdapTree/varscan_pipeline: Publication release (Version 1.0.0). Zenodo. http://doi.org/10.5281/zenodo.5083302
    ```

1. [CoAdapTree CMH test](https://github.com/brandonlind/cmh_test) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083798.svg)](https://doi.org/10.5281/zenodo.5083798)
    - This code takes output from the VarScan pipeline and calculates Cochran-Mantel-Haenszel chi-squared tests on stratified contingency tables of read counts standardized by population ploidy.
    ```
    Lind B (2021) GitHub.com/brandonlind/cmh_test: preprint release (Version 1.0.0). Zenodo. http://doi.org/10.5281/zenodo.5083798
    ```

1. [Douglas-fir GEA populations (in progress)](https://github.com/brandonlind/douglas_fir_natural_populations) 
    - This repository contains the processessing scripts needed to analyze range-wide populations of Douglas-fir.
    - Lind BM. 2023. GitHub.com/brandonlind/douglas_fir_natural_populations: Offset Revision 1 (v1.0.0). Zenodo. https://doi.org/10.5281/zenodo.8018894

1. [Jack pine GEA populations (in progress)](https://github.com/brandonlind/jack_pine_natural_populations/blob/main/README.md)
    - This repository contains the processessing scripts needed to analyze range-wide populations of jack pine.
    - Lind BM. 2023. GitHub.com/brandonlind/jack_pine_natural_populations: Offset Revision 1 (v1.0.0). Zenodo. https://doi.org/10.5281/zenodo.8018892

1. [CoAdapTree Douglas-fir / lodgepole pine SNP array (in progress)](https://github.com/brandonlind/coadaptree_snparray)
    - This repository covers the code used to develop a 50k SNP array split evenly between Douglas-fir and lodgepole pine.

-----
### Publications and associated repositories
1. (2021) <b> BM Lind* </b>, M Lu*, D Vidakovic, P Singh, T Booker, S Yeaman, S Aitken. Haploid, diploid, and pooled exome capture recapitulate features of biology and paralogy in two non-model tree species. Molecular Ecology Resources. [Direct Link](https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13474) or [Download](pdfs/Lind_et_al_2021-Haploid-diploid-and-pooled-exome-capture-recapitulate-features-of-biology-and-paralogy-in-two-non-model-tree-speces.pdf)
    - <b> Abstract </b>
Despite their suitability for studying evolution, many conifer species have large and repetitive giga-genomes (16-31Gbp) that create hurdles to producing high coverage SNP datasets that captures diversity from across the entirety of the genome. Due in part to multiple ancient whole genome duplication events, gene family expansion and subsequent evolution within Pinaceae, false diversity from the misalignment of paralog copies create further challenges in accurately and reproducibly inferring evolutionary history from sequence data. Here, we leverage the cost-saving benefits of pool-seq and exome-capture to discover SNPs in two conifer species, Douglas-fir (Pseudotsuga menziesii var. menziesii (Mirb.) Franco, Pinaceae) and jack pine (Pinus banksiana Lamb., Pinaceae). We show, using minimal baseline filtering, that allele frequencies estimated from pooled individuals show a strong positive correlation with those estimated by sequencing the same population as individuals (r > 0.948), on par with such comparisons made in model organisms. Further, we highlight the use of haploid megagametophyte tissue in identifying sites that are likely due to misaligned paralogs. Together with additional minor filtering, we show that it is possible to remove many of the loci with large frequency estimate discrepancies between individual and pooled sequencing approaches, improving the correlation further (r > 0.973). Our work addresses bioinformatic challenges in non-model organisms with large and complex genomes, highlights the use of megagametophyte tissue for the identification of paralog sites when sequencing large numbers of populations, and suggests the combination of pool-seq and exome capture to be robust for further evolutionary hypothesis testing in these systems.
    - [Code for manuscript](https://github.com/brandonlind/testdata_validation) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083292.svg)](https://doi.org/10.5281/zenodo.5083292)
    - [Data for manuscript](https://doi.org/10.5061/dryad.k0p2ngf7w)
[![DOI](https://zenodo.org/badge/DOI/10.5061/dryad.k0p2ngf7w.svg)](https://doi.org/10.5061/dryad.k0p2ngf7w)

1. (2022) P Singh, B St Clair, <b> BM Lind </b>, R Cronn, NP Wilhelmi, M Lu, D Vidakovic, R Hamelin, D Shaw, S Aitken, S Yeaman Genomic architecture of resistance and tolerance to Swiss needle cast and Rhabdocline needle cast diseases in Douglas-fir. EcoEvoRxiv https://doi.org/10.32942/osf.io/s5rd7
    - <b> Abstract </b>
Understanding the genetic architecture of tolerance and resistance to pathogens is important to monitor and maintain resilient tree populations. Here we investigate the genetic basis of tolerance and resistance and to needle cast disease in Douglas-fir (Pseudotsuga menziesii) caused by two fungal pathogens: Swiss needle cast (SNC) caused by Nothophaeocryptopus gaeumannii, and Rhabdocline needle cast (RNC) caused by Rhabdocline pseudotsugae). We performed a case-control genome-wide association analysis (GWAS) and found these traits to be polygenic. Significant associations with SNC resistance were found for SNPs in genes for stomatal regulation and ethylene and jasmonic acid pathways, which are known for their roles in plant defense and immunity. Top-associated SNPs for SNC tolerance were found in genes of secondary metabolite pathways. We identified a key upstream transcription factor of plant defence, ERF1, as the main candidate for RNC resistance. Our findings contribute to the understanding of the highly polygenic architectures underlying disease resistance and tolerance in Douglas-fir and have important implications for forestry and conservation as the climate changes.
    - Code for Manuscript [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083798.svg)](https://doi.org/10.5281/zenodo.5083798)

1. (2023) <b> BM Lind </b>, R Candido-Ribeiro, P Singh, M Lu, DO Vidakovic, TR Booker, M Whitlock, N Isabel, S Yeaman, and SN Aitken.How useful is genomic data for predicting¬† maladaptation to future climate? biorxiv, doi: 10.1101/2023.02.10.528022.
    - <b> Abstract </b>
Methods using genomic information to forecast potential population maladaptation to climate change are becoming increasingly common, yet the lack of model validation poses serious hurdles toward their incorporation into management and policy. Here, we compare the validation of two methods ‚Äì Gradient Forests (GF) and the Risk Of Non-Adaptedness ‚Äì using exome capture pool-seq data from 35 to 39 populations across three conifer taxa: two Douglas-fir varieties and jack pine. We evaluate sensitivity of these algorithms to the source of input loci (markers selected from genotype-environment associations [GEA] or those selected at random). We validate these methods against two-year and 52-year growth and mortality measured in independent transplant experiments. Overall, we find that both methods often better predict transplant performance than climatic or geographic distances. We also find that while GF and RONA models are surprisingly not improved using GEA candidates. Even with promising validation results, ambiguity of model projections to future climates makes it difficult to identify the most maladapted populations using either method. Our work advances understanding of the sensitivity and applicability of these approaches, and we discuss recommendations for their future use.
    - Code for manuscript [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5083292.svg)](https://doi.org/10.5281/zenodo.7641225) [![DOI](https://zenodo.org/badge/651143319.svg)](https://zenodo.org/badge/latestdoi/651143319) [![DOI](https://zenodo.org/badge/651215149.svg)](https://zenodo.org/badge/latestdoi/651215149)
    
